CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration.uc

```diff
@@ -0,0 +1,80 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  NotSoLoneWolf
+//  PURPOSE: This class contains a variety of tools
+//           for this mod and others to interact with
+//           the infiltration template system.
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+
+class X2Helper_Infiltration extends Helpers config(Infiltration);
```

Comment:
Why `Helpers`? Extend `Object` until you have a good reason not to.

Reply:
Yeah, I looked at `Helpers`, there is nothing we really need there (and if we do, then we just use `class'Helpers'.static.Whatever()`)